res/database/Default/user/stukov_ads.xml

```diff
@@ -923,6 +1014,7 @@
 			<description>
 				<de>Lebensläufe? Duftkerzen? Aubert!</de>
 				<en>Resumes? Scented candles? Aubert!</en>
+				<pl>Świece zapachowe? Aubert!</pl>
```

Comment:
Lebensläufe / Resumes here means means Curriculum vitae (in plural). C.V. doesn't quite swing here, maybe you have another good term in Polish for that?